src/utils/ethereumUtils.ts

```diff
@@ -401,7 +429,7 @@ const deriveAccountFromPrivateKey = privateKey => {
   };
 };
 
-const deriveAccountFromWalletInput = input => {
+const deriveAccountFromWalletInput = (input: any) => {
```

Comment:
I think `input` here is a `string`, or more accurately a `EthereumWalletSeed` from `src/model/wallet.ts`, which resolves to a string.